extension/chrome/elements/pgp_block_modules/pgp-block-signature-module.ts

```diff
@@ -55,9 +55,9 @@ export class PgpBlockViewSignatureModule {
           } catch (e) {
             if (ApiErr.isSignificant(e)) {
               Catch.reportErr(e);
-              $('#pgp_signature').addClass('bad').find('.result').text(`Could not load sender's pubkey due to an error.`);
+              $('#pgp_signature').addClass('red_label').find('.result').text(`error verifying signature: ${e}`);
             } else {
-              $('#pgp_signature').addClass('bad').find('.result').text(`Could not look up sender's pubkey due to network error, click to retry.`).click(
+              $('#pgp_signature').addClass('red_label').find('.result').text('error verifying signature: offline, click to retry').click(
                 this.view.setHandler(() => window.location.reload()));
```

Comment:
@tomholub on retry we reload the whole frame, should we make an effort to preserve the rendered text?

Reply:
It would be nicer, but not really necessary. 

In this case, user has actively clicked on something, so they should expect things to change. 

That's different than when an automated process changes things when they thought it's already done.